### ISSUE TITLE
Show name of the program

### ISSUE THREAD

**Issue by bbKaNaTdd** [tjackenpacken/taskbar-groups#217]
It would be nice to see the name of the program when the mouse cursor  over the shortcuts.

**Comment by roselovesgreene** [comment]
agreed. i've been using this for organizing chapters for stories and it's annoying to have to either:

1. open each individual file and see if it's the right one.
2. create a shortcut for each and every file that has a custom icon with the chapter number.

this would be _sooo_ helpful to have

**Comment by bbKaNaTdd** [comment]
not being able to see shortcut names is a big handicap

**Comment by citsuresh** [comment]
I desperately need this. 
Can we add custom tooltip text to the shortcuts in the group?
The tooltip text can be the program name by default. The user could change it.

In my case I have all the shortcuts with the same icon and I cannot figure out which shortcut launches what. Right now, I am forced to use a custom icon for each shortcut to indicate what it does.

**Comment by gaveitatry2** [comment]
I downloaded TaskbarGroups v0.2.0.0 today on Feb 10, 2023 for Windows 10. I wanted to use this program, but unfortunately I could not. I tested the program out by adding 11 shortcuts. Each shortcut was to a Ren'Py Visual Novel game. And every shortcut had the same icon. When I pinned the taskbar group to the taskbar, I was sad to see that I could not distinguish any of the 11 shortcuts from each other because there was no tool tip to tell me the shortcut name. The only workaround I can see for this is to create custom icons for each shortcut but that's too much work and headache. Hopefully a future version of this program will have shortcut program names on tooltips when you hover over the shortcuts.
